jms/serializer-bundle/2.0/manifest.json

```diff
@@ -3,6 +3,7 @@
         "JMS\\SerializerBundle\\JMSSerializerBundle": ["all"]
     },
     "copy-from-recipe": {
-        "config/": "%CONFIG_DIR%/"
+        "config/": "%CONFIG_DIR%/",
+        "metadata/": "metadata/"
```

Comment:
Will this always work? Don't we need a prefix?

Reply:
I guess it will always work, because `%CONFIG_DIR% === 'config'`